### PR TITLE
18 creation of main view

### LINKED ISSUE #18: Creation of Main view
# REQUIREMENTS
Elaboration of main view to do checkIn/checkput  
Create feature to do checkIn / checkOut
# DESIGN
imagen

### PR DESCRIPTION
**Pull request by Javi1499**
# Description
What did you implemented/Why did you implemented this:
 -What? I added main view where employees will done check in/out. It incluide controller, new view and changes in model and implementation of flash messages
 - Why? Flash Messages was implemented  to return feedback to user about the action
 
  ## Screenshots
imagen

### CODE REVIEW

File: app/controllers/records_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      flash[:success] = "CheckOut at #{time.to_formatted_s(:short)}. Have a good night "
+    else
+      Record.create! employee_id: id_employee.id, check_in: time
+      flash[:success] = "CheckIn at #{time.to_formatted_s(:short)}. Have a nice day"
+    end
+    redirect_to action: :new
+  rescue StandardError => e
+    flash[:danger] = 'Private number not exist. Please try again'
+    redirect_to action: :new
+  end
```
Comment (vyk2rr) [inline app/controllers/records_controller.rb]: this logic is a bit hard to understand. I would move it to the model where it belongs and keep the controller skinny, quick example => https://riptutorial.com/ruby-on-rails/example/9609/fat-model--skinny-controller

Comment (Javi1499) [reply app/controllers/records_controller.rb]: Thanks. I changed my code using this format

File: app/helpers/records_helper.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module RecordsHelper
+
+end
```
Comment (YilKanda) [inline app/helpers/records_helper.rb]: If you don't use this helper, you can remove it.

**Review by YilKanda — DISMISSED** [review]

**Review by femm34 — APPROVED** [review]
LGTM :)

**Review by germanruzca — APPROVED** [review]
LGTM

**Review by EliasUR — APPROVED** [review]
LGTM

**Pull request merged**